pt-BR/active_record_validations.md

```diff
@@ -870,12 +870,12 @@ end
 
 ### `:on`
 
-The `:on` option lets you specify when the validation should happen. The
-default behavior for all the built-in validation helpers is to be run on save
-(both when you're creating a new record and when you're updating it). If you
-want to change it, you can use `on: :create` to run the validation only when a
-new record is created or `on: :update` to run the validation only when a record
-is updated.
+A opção `:on` permite que você espeficique quando a validação deve acontecer. O
+comportamento padrão para todos os `helpers` de validação `built-in` é ser executado
```

Comment:
```suggestion
comportamento padrão para todos os *helpers* de validação `built-in` é ser executado
```

Reply:
Talvez podemos traduzir `build-in` para embutidos, integrado, incluso ou algo assim o que acha?

Reply:
acho interessante **integrados** nesse caso